content/docs/concepts/clusters.mdx

```diff
@@ -0,0 +1,205 @@
+   ```yaml
+   apiVersion: v1
+   kind: Secret
+   metadata:
+     name: pomerium
+     namespace: pomerium-zero
+   type: Opaque
+   stringData:
+     pomerium_zero_token: <cluster_token>
+   ```
+1. Apply the manifest to your Kubernetes cluster
+   ```bash
+   kubectl apply -f ./secret.yaml
+   ```
+
+</TabItem>
+</Tabs>
+
+### Cluster domain
+
+Pomerium Zero generates and assigns a unique starter **Cluster Domain** to your cluster. A cluster domain is a randomly generated fully qualified domain name that follows this format:
+
+`<CLUSTER-SUBDOMAIN-XXXX>.pomerium.app`
+
+For example: `voracious-ape-1578.pomerium.app`
+
+The starter cluster domain comes with its own DNS records and TLS certificates so you can quickly test Pomerium Zero before adding your own custom domain.
+
+:::info
+
+See the **Custom Domains** capabilities page for more information on how to add custom domains to a cluster.
+
+:::
+
+#### Cluster name
+
+The **Cluster Name** is a customizable identifier for your cluster. It defaults to the randomly generated subdomain of your cluster domain, but you can change it at any time. Changing the cluster name does not affect your cluster domain.
+
+If you want to change the cluster name, you can update it in your cluster settings:
+
+1. From the navigation bar dropdown, select **Manage Clusters**
+1. Under **Actions**, select the **edit** icon
+1. In the **Cluster Name** field, enter your preferred cluster name
+1. Save your changes and apply the changeset
+
+![Viewing the new cluster name in the Zero Console](./img/clusters/cluster-name-table.png)
+
+#### Detected and Override IP Address
+
+Whenever you start a cluster, it must fetch its configuration from the Pomerium Zero control plane. During this process, Pomerium attempts to detect the cluster's outbound IP address to connect to it.
+
+In the Zero Console, the cluster's IP address is referred to as the **Detected IP Address**. Depending on your environment, the detected IP address may not be reachable by you, the end-user. This could be the case if, for example, you're running a cluster:
+
+- Locally from your laptop behind a NAT-enabled router
+- Inside a corporate intranet
+- Behind a firewall
+- In a Kubernetes server where the ingress load balancer IP address doesn't match the egress IP address
+
+In such cases, you may see a `DNS_PROBE_FINISHED_NXDOMAIN` error in your browser when you access a route. As a workaround, we've provided the **Override IP Address** field, which allows you to manually set the IP address so that Pomerium Zero can connect to your cluster.
+
+![Manually setting the cluster IP address in the Zero Console](./img/clusters/cluster-ip-address-settings.png)
+
+### Cluster settings
+
+Each cluster is assigned its own **Cluster settings** object, meaning the available settings within a cluster apply only to that cluster instance. This means changes applied to one cluster instance won't affect another cluster deployed with the same token.
```

Comment:
```suggestion
Each cluster is assigned its own **Cluster settings** object, meaning the available settings within a cluster apply only to that cluster instance. This means changes applied to one cluster instance won't affect another cluster deployed with the same token.
```

this is incorrect. all clusters deployed with the same token share all the configuration (settings, routes, certs, etc).